int8 robot_dofs
float32[] traj  
